src/writing-policies/spec/06-signature-verifier-policies.md

```diff
@@ -0,0 +1,147 @@
+# Signature verifier policies
+
+Kubewarden implements support for the [Sigstore](https://www.sigstore.dev/)
+standard. This allows to implement a Secure Supply Chain for your cluster.
+
+Part of that is to ensure that all container images run the in the cluster are
+signed and verified, proving that they come from their stated authors, and
+haven't been tampered with (for other concerns, see the docs on [how we implement a Secure Supply Chain for the policies themselves](TODO link to https://github.com/kubewarden/docs/pull/100/files)).
+
+Sigstore signatures of containers are stored as layers of the container image.
+Policies that want to check Sigstore signatures of containers need then to check
+those container layers, and would need to pull the signature layers to see the
+signatures themselves.
+
+Obtaining and operating with those container layers needs to happen outside of
+the waPC guest (the policy), and on the waPC host (Kubewarden's
+`policy-server`).
+
+Just like for context-aware policies, the guest-host intercommunication is
+performed using the regular waPC host calling mechanism, and any guest
+implementing the waPC intercommunication mechanism is able to request this
+information.
```

Comment:
Ok, here we mention waPC again. We can leave it or drop the whole paragraph, after all the ma "magic" is done by the SDK and is not something relevant for the end user.

Reply:
indeed, done in https://github.com/kubewarden/docs/pull/109/commits/7b6920fa261a7c92632e084b3532db7215f61ada